File: 3.c
#include <stdio.h>
#include <string.h>

int main()
{
    char* name[20]

    printf("What is your name?");

    gets(name);

    if (strcmp(name, "Alice") == 0)
    {
        printf("Hello %s", name);
    } else
    {
        printf("Hello stranger!");
    }
    
    return 0;

}

File: week 2/3.py
# Question 3: Modify the previous program such that only the users Alice and Bob are greeted with their names.

# taken from: https://adriann.github.io/programming_problems.html


name = input("What is your name?")

if (name == "ALice") or (name == "Bob"):
    print("Hello %s, How are you?" %(name))

else:
    print("Hello stranger!")




name = input("What is your name?")

if (name=="Bob") or (name=="Alice"):
    print("Hello %s, How are you?" %(name))
else:
    print("Hello lowly peasant!")